tests/imposter/data/questions.json

```diff
@@ -1,10 +1,12 @@
 {
-  "questions": [
-    {
-      "questionKey": "rti-payslip-national-insurance",
-      "info": {
-        "months": "3"
+  "questions": {
+    "single-question": [
+      {
+        "questionKey": "rti-payslip-national-insurance",
+        "info": {
+          "months": "3"
+        }
       }
-    }
-  ]
+    ]
```

Comment:
Nice, so we'll be able to define different sets of questions in here one for payslips one for P60s etc?

Reply:
Yep, we can add in any custom journey/set of questions we want as a sub-group in here, and then all we need to do is add a new set of journey apis in private-api-config with a session-id that matches the question group name